modules/lib/lib-event/README.md

```diff
@@ -0,0 +1,50 @@
+# Enonic XP lib-event TS types
+
+> TypeScript definitions for `lib-event` library of Enonic XP
+
+## Install
+
+```bash
+npm i --save-dev @enonic/lib-event
+```
+
+## Use
+
+__1. Update tsconfig.json__
+
+Add the corresponding types to your `tsconfig.json` file that is used for application's server-side TypeScript code:
+
+```json
+{
+  "compilerOptions": {
+    "types": [
+      "@enonic/lib-event"
+    ]
+  }
+}
+```
+
+### Global XP types
+
+You may also want to add the `@enonic/script-impl` types, since it provides a definitions for XP global objects, e.g. `log`, `app`, etc. The
+documentation can be found [here](https://github.com/enonic/xp/tree/master/modules/script/script-impl/README.md).
+
+## Example
+
+```typescript
+const eventLib, {EnonicEvent} = __non_webpack_require__('/lib/xp/event');
```

Comment:
`__non_webpack_require__` is heavily dependent on webpack. We must support not only webpack

Reply:
You are correct, it's a webpack-related stuff. Will replace it with `require`. 
Since we may want to keep code more generic, in the `base` (`script-impl` types), I'll remove the `__non_webpack_require__` from global definitions, and add it as an example/note in the readme instead. I'll also move the actual code to `starter-webpack`.